### DISCUSSION TITLE
is there a way to change the design?

### DISCUSSION THREAD

**Discussion by bloodyeyesblackblood** [deckerst/aves#749] [General]
i use aves and simple gallery, i like the design of simple gallery more than aves.. 
but aves seems to more useful and somehow a little nicer with it's viewer, sadly the border radius annoys me on albums :(

didn't find a way to disable border radius, only border color? 
so is there maybe a theme switcher for community based themes or can i make my own in-app?
or can i atleast disable border raidus somewhere? 

i would also like to switch the font, but it isn't really important yet.

would like to completely switch over.

EDIT: found a issue where it's also discussed that deckerst won't remove the borders? but whats with togglin' the border or changin' the radius at least?

**Comment by deckerst** [comment]
At the moment there is no way to do so, and it is unlikely to change. Having a distinct look is actually one of the goals with Aves.

Besides, curved borders indicate filters throughout the app (an album is one of many kinds of filters). It's consistent that way.

**Reply by bloodyeyesblackblood** [reply]
oh alright :( 
sad but understandable
